long-ride-alerts/src/main/java/org/apache/flink/training/exercises/longrides/LongRidesExercise.java

```diff
@@ -67,7 +67,7 @@ public static void main(String[] args) throws Exception {
 		DataStream<TaxiRide> rides = env.addSource(rideSourceOrTest(new TaxiRideSource(input, maxEventDelay, servingSpeedFactor)));
 
 		DataStream<TaxiRide> longRides = rides
-				.keyBy(ride -> ride.rideId)
+				.keyBy((TaxiRide ride) -> ride.rideId)
```

Comment:
I believe that this type definition is not strictly necessary. Is it added to make it easier for comprehension?

Reply:
That's right. It's not necessary, I just added it for clarity.